boards/riscv/rv32m1_vega/Kconfig.defconfig

```diff
@@ -68,4 +69,80 @@ config I2C_3
 
 endif # I2C
 
+
+if BT
+
+config BT_CTLR
+	default y
+
+choice BT_LL_CHOICE
+	default BT_LL_SW_SPLIT
+endchoice
+
+config RV32M1_INTMUX_CHANNEL_2
+	default n
+
+config RV32M1_INTMUX_CHANNEL_3
+	default n
+
+#
+# Block of overrides. Need to work through these to determine what should be
+# an override
+#
+# Enable support for Bluetooth 5.0 PHY Update Procedure.
+config BT_PHY_UPDATE
+	default n
+
+# Enable support for Bluetooth v4.2 LE Data Length Update procedure.
+config BT_DATA_LEN_UPDATE
+	default n
+
+# Zephyr HCI Vendor-Specific Commands
+config BT_HCI_VS
+	default n
+
+# Overrides of subsys/bluetooth/controller/Kconfig:
+config BT_CTLR_CRYPTO
+	default y
+config BT_CTLR_LE_ENC
+	default n
+config BT_CTLR_CONN_PARAM_REQ
+	default n
+config BT_CTLR_EXT_REJ_IND
+	default n
+config BT_CTLR_SLAVE_FEAT_REQ
+	default n
+config BT_CTLR_LE_PING
+	default n
+config BT_CTLR_PRIVACY
+	default n
+config BT_CTLR_EXT_SCAN_FP
+	default n
+config BT_CTLR_CHAN_SEL_2
+	default n
+config BT_CTLR_ADV_EXT
+	default n
+# Odd one to turn on?
+config BT_CTLR_ADVANCED_FEATURES
```

Comment:
nRF specific Kconfig could be under depends. But this can be done as port of new PR

Reply:
OK, we can do it a new PR